kanidm_tools/src/cli/account.rs

```diff
@@ -89,12 +90,33 @@ impl AccountOpt {
                 }
                 AccountRadius::Delete(aopt) => {
                     let client = aopt.copt.to_client().await;
-                    if let Err(e) = client
+
```

Comment:
Is this allowing json output for the cli?

Reply:
Yep

Reply:
Okay, I think for now, I'm not sure about the json here. Can we remove that, because I think this get's too close to "cli as an api" and we got absolutely burnt by that in other projects.

Otherwise I appreciate the ideas to improve / reduce the boilerplate-ness. Beside that it all looks good!

Reply:
I went another way - forced output mode to text, removed the user option and retained the automatic message handler 😄